Add GodotCon 24 Review (#953)
godotengine/godot-website: collections/_article/review-godotcon24.md

What do you do at a conference, besides watching talks, browsing through the exhibition, or doing business? Have some fun and get to know other attendees in a casual way, of course.

This is exactly what we aimed to facilitate with our onsite activities, especially the [sticker challenge](https://conference.godotengine.org/2024/activities/) tying them all together (since many of you asked: the stickers were designed by community artist [Tibo](https://gotibo.fr/)). From networking bingo to graffiti canvas, we saw many people striking up conversations with strangers or having a great time with friends.

<img alt="People participating in a game jam" src="/storage/blog/godotcon24/one-pc-jam.jpg"/>

One of the most popular activities turned out to be the One PC Jam (taking place at three laptops concurrently) where the conference attendees created a jam game over the course of two days. The twist: everyone only was granted one hour on the project, before having to concede their seat to the next person. This resulted in three separate, actually playable games, which you can [find here](https://godotcon.itch.io/).

# Talks & Workshops

> The Godot community once again came through and submitted more interesting talks than we could fit on the program - resulting in us repurposing the admittedly rather small workshop room to squeeze some more of the talented speakers in. Luckily, we also invited a camera crew onsite, that took great care recording both the workshops and two talk tracks, for you to enjoy on the Godot Engine YouTube channel - some of them already available, the rest being edited as you read this blog post.

Author: ```suggestion

<img alt="View of the stage, with people standing behind full rows of chairs." src="/storage/blog/godotcon24/backstage.jpg"/>

The Godot community once again came through and submitted more interesting talks than we could fit on the program - resulting in us repurposing the admittedly rather small workshop room to squeeze some more of the talented speakers in. Luckily, we also invited a camera crew onsite, that took great care recording both the workshops and two talk tracks, for you to enjoy on the Godot Engine YouTube channel - some of them already available, the rest being edited as you read this blog post.
```

---
title: "GodotCon 24: Berlin Edition"
excerpt: "Our biggest conference yet!"
categories: ["events"]
author: Nat
image: /storage/blog/covers/godotcon-2024-post.webp
date: 2024-11-11 18:30:00
---

<br>

> 📋 [Feedback form for visitors](https://forms.gle/3tALsQid32qqvu2Z8)

Reviewer: Are these intended to appear on separate lines?

Reviewer 2: seems like it, but I like that they don't take too much space, so I think we can leave as is

🎤 [Talk & workshop recordings](https://www.youtube.com/playlist?list=PLeG_dAglpVo6TS0q858NajyeglRuvb7hs)
🫙 [Jam games](https://godotcon.itch.io/)
<hr>

Variety was the name of the game this year. GodotCon keeps going around the globe and adds more activities to the program: the event team made up of volunteers and Godot Foundation staff put a lot work into making this more than a mere meetup occasion for the ever-growing community forming around the Godot Engine. 

> Judging by the smiley pictures taken all around the [Silent Green](https://www.silent-green.net/) venue, and the excited chatter about what the next installments might bring to the table, we seem to have hit the sweet spot for our 574 attendees. For those of you that couldn’t be there yourself - considering both ticket batches sold out within an hour - join us on a retrospective walk through GodotCon, as well as highlighting all the wonderful speakers, exhibitors, and sponsors that made this event possible.

Reviewer: ```suggestion
Judging by the smiley pictures taken all around the [Silent Green](https://www.silent-green.net/) venue, and the excited chatter about what future events might bring to the table, we seem to have hit the sweet spot for our 574 attendees. For those of you that couldn’t be there yourself - considering both ticket batches sold out within an hour - join us on a retrospective walk through GodotCon, highlighting all the wonderful speakers, exhibitors, and sponsors that made this event possible.
```
"as well as" seemed unnecessary.

I'm not sure about "next installments" - did you mean future events or future versions of Godot Engine?
EDIT: Thanks, have edited my suggestion.

Reviewer 2: I think it is future versions of the GodotCon

# Sponsor support
GodotCon 2024 wouldn’t have been possible without the generosity of [our sponsors](https://conference.godotengine.org/2024/#sponsors), who enabled us to bring this event to life.

## Critical Reflex

> Having a publisher with experience in Godot on-board was as exciting for us as it seemed for the audience of “Indie Publishing 101” by their business developer Daniel. Additionally, you might have spotted the goodie bags they provided us with.

Reviewer: ```suggestion
Having a publisher with experience in Godot on-board was as exciting for us as it seemed for the audience of “Indie Publishing 101” by their business developer Daniel. Additionally, you might have spotted the goodie bags they provided us.
```
To avoid ending the sentence with a preposition!

🔗 [criticalreflex.com](https://criticalreflex.com)

<img alt="Venue picture featuring Critical Reflex goodie bag" src="/storage/blog/godotcon24/goodie-bag.webp"/>

## Endless Foundation
“Block Coding: Making Godot Accessible to Learners Everywhere” was another well-visited talk on the agenda, as was the booth of the biggest exhibitor team onsite. Their tech demonstration drew in fellow educators as well as curious game developers looking to contribute.

🔗 [endlessos.org](https://www.endlessos.org/)

<img alt="Screenshot of their block coding editor" src="/storage/blog/godotcon24/block-coding.webp"/>

## W4 Games

> Port your Godot games to console with ease! The company founded by Godot Engine veterans explained “W4's role in the Godot Ecosystem” on stage and let you playtest their products & talk business at their busy booth. 

Reviewer: ```suggestion
Port your Godot games to console with ease! The company founded by Godot Engine veterans explained “W4's role in the Godot Ecosystem” on stage and let you playtest their products and talk business at their busy booth. 
```

🔗 [w4games.com](https://w4games.com)

> <img alt="" src="/storage/blog/godotcon24/w4-games.webp"/>

Author: ```suggestion
<img alt="Juan giving a presentation about W4 Games on stage" src="/storage/blog/godotcon24/w4-games.webp"/>
```

> <img alt="" src="/storage/blog/godotcon24/w4-games.webp"/>

Reviewer: Needs alt text.

Reviewer 2: cc @MurderVeggie

## Poki

> The provider of free online games also has shown interest in Godot before and is hoping to expand their selection with your projects. We talked with them about improving Godot for the Web together, let’s see what the future might bring!

Reviewer: ```suggestion
The provider of free online games also has shown interest in Godot before and is hoping to expand their selection with your projects. We talked with them about improving Godot for the Web together; let’s see what the future might bring!
```
Needs a bigger break than just a comma. Alternatively, if you want the comma, need to add a word after it like "so".

🔗 [poki.com](https://poki.com)

> <img alt="" src="/storage/blog/godotcon24/poki.webp"/>

Author: ```suggestion
<img alt="Photo of the Poki booth with two of the representatives smiling and giving a thumbs up" src="/storage/blog/godotcon24/poki.webp"/>
```

> <img alt="" src="/storage/blog/godotcon24/poki.webp"/>

Reviewer: Needs alt text.

Reviewer 2: cc @MurderVeggie

## AWS

> After demonstrating “How to quick start a Godot Engine game on AWS” the Cloud provider team spent time answering questions on the exhibition floor.

Reviewer: ```suggestion
After demonstrating “How to quick start a Godot Engine game on AWS”, the cloud provider team spent time answering questions on the exhibition floor.
```

🔗 [aws.amazon.com](https://aws.amazon.com/gametech/)

> <img alt="" src="/storage/blog/godotcon24/aws.webp"/>

Reviewer: Needs alt text.

<hr>

If you or your company are interested in supporting the next edition of GodotCon, we’d love to hear from you! <br>Please reach out to us at [godotcon@godotengine.org](mailto:godotcon@godotengine.org).

<hr>

> # Game Exhibition

Reviewer: ```suggestion
# Game exhibition
```
Generally I believe we use "Sentence case" for headings within articles.

- [Robot Detour](https://store.steampowered.com/app/2666840/Robot_Detour/)
- [Skye's Ducky Adventure: Quest for Gluttony](https://schwarzeralptraum.itch.io/skyes-ducky-adventure-quest-for-gluttony)
- [Voice of Flowers](https://store.steampowered.com/app/2609560/Voice_of_Flowers/)
- Window Arcade

# Activities

<img alt="The graffiti canvas full of different drawings by the visitors" src="/storage/blog/godotcon24/canvas.webp"/>

What do you do at a conference, besides watching talks, browsing through the exhibition, or doing business? Have some fun and get to know other attendees in a casual way, of course.

This is exactly what we aimed to facilitate with our onsite activities, especially the [sticker challenge](https://conference.godotengine.org/2024/activities/) tying them all together (since many of you asked: the stickers were designed by community artist [Tibo](https://gotibo.fr/)). From networking bingo to graffiti canvas, we saw many people striking up conversations with strangers or having a great time with friends.

<img alt="People participating in a game jam" src="/storage/blog/godotcon24/one-pc-jam.webp"/>

> One of the most popular activities turned out to be the One PC Jam (taking place at three laptops concurrently) where the conference attendees created a jam game over the course of two days. The twist: everyone only was granted one hour on the project, before having to concede their seat to the next person. This resulted in three separate, actually playable games, which you can [find here](https://godotcon.itch.io/).

Reviewer: ```suggestion
One of the most popular activities turned out to be the One PC Jam (taking place at three laptops concurrently) where the conference attendees created a jam game over the course of two days. The twist: everyone only was granted one hour on the project, before having to concede their seat to the next person. This resulted in three separate, actually playable games, which you can [find on our GodotCon itch.io site](https://godotcon.itch.io/).
```
Generally best to avoid "here" in links.

> # Talks & Workshops

Reviewer: ```suggestion
# Talks and workshops
```

<img alt="View of the stage, with people standing behind full rows of chairs." src="/storage/blog/godotcon24/backstage.webp"/>

The Godot community once again came through and submitted more interesting talks than we could fit on the program - resulting in us repurposing the admittedly rather small workshop room to squeeze some more of the talented speakers in. Luckily, we also invited a camera crew onsite, that took great care recording both the workshops and two talk tracks, for you to enjoy on the Godot Engine YouTube channel - some of them already available, the rest being edited as you read this blog post.

> <span style="font-size: 1.5em">

Reviewer: Just a personal opinion, but I dislike the larger text used here.

    <div>
        <input type="hidden" name="nonce" />
        <input type="email" name="email" required placeholder="E-mail"  style="font-size: 18px;padding: 16px 18px;border: 1px solid var(--base-color);box-shadow: 0 4px 9px -3px #0000002e;background: var(--card-footer-color);color: var(--base-color-text);"/>
        <p style="display:none;">
          <input id="6e6e9" type="checkbox" name="l" checked value="6e6e9ba4-732b-4528-b4cd-71bbe6850f8d" />
          <label for="6e6e9">Godot Conference</label>
        </p>
				<input type="submit" class="btn" value="Subscribe" style="cursor: pointer;border: inherit; border-radius: 0;
  box-shadow: none;
  height: 54px;" />
    </div>
</form>

🌐 [https://conference.godotengine.org/](https://conference.godotengine.org/) 

### 👋 See you next year!

<style>
	.preview-image-container {
		display: flex;
		justify-content: center;
		align-items: center;
		gap: 5px;
		/* padding: 16px; */
		background-color: transparent;
	}
	.preview-image-container {
		background-color: transparent;
	}
	.preview-image-container img {
		max-height: 300px;
		background-color: transparent;
	}

> </stlye>

Reviewer: ```suggestion
</style>
```
